product-service/src/handlers/postProduct.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  APIGatewayProxyEvent,
+  APIGatewayProxyHandler,
+  APIGatewayProxyResult,
+} from 'aws-lambda';
+import 'source-map-support/register';
+import Joi from 'joi';
+import { Client } from 'pg';
+import dbOptions from '../dbConfig';
+
+console.log('DB CONFIG::', dbOptions);
+
+const postProduct = ` 
+  with ans1 as (
+    insert into products(title, description, price) 
+    values($1, $2, $3)
+    returning id
+  )
+  insert into stocks (product_id, count)
+  values (
+    (select id from ans1), $4
+  )
+  returning product_id;
+ `;
+
+const defaultHeaders = {
+  'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+  'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+};
+
+const requestSchema = Joi.object({
+  count: Joi.number().integer().min(0).required(),
+  description: Joi.string().required(),
+  price: Joi.number().integer().min(0).required(),
+  title: Joi.string().required(),
+});
```

Comment:
FE uses Yup and I assume you could reuse it, what is the motivation for using Joi?

Reply:
Probably you're right, I'll consider refactoring during next task